### PR TITLE
Release Proposal: v2.13

### PR DESCRIPTION
**Pull request by mattmcneeney**
This triggers the start of the [Review Process](https://github.com/openservicebrokerapi/servicebroker/blob/master/CONTRIBUTING.md#review-process)

---

**edit by @avade**

## Review is open until Tuesday 26th September

### CODE REVIEW

**Comment by angarg12** [comment]
Travis is complaining about links, but I guess it is unavoidable since those URLs won't exists until the PR is merged, is that right?

File: release-notes.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-# Service Broker API Release Notes #
-
-## <a id='2-12'></a>v2.12 ##
+# Service Broker API Release Notes
+
+## v2.13
```
Comment (avade) [inline release-notes.md]: I know we don't have this for previous releases, but links from the feature to the relevant part of the spec would be very helpful. I can do this post merge.

File: spec.md [hunk 2]
```diff
@@ -85,24 +85,28 @@ added to enable support for new features.
...
-* The specification now uses RFC2119 keywords to make it clearer when things are REQUIRED rather than OPTIONAL or RECOMMENDED.
-* Request and response parameters of type string, if present, MUST be a non-empty string.
-* Cleaned up text around status codes in responses, clarifying when they MUST be returned.
-* Added clarity around the `app_guid` request field.
-* Clarified the semantics of `plan_id` and `parameters` fields in the updating a service instance request body.
-* Clarified the default value of the `plan_updateable` field.
-* Clarified when `route_service_url` is REQUIRED and when brokers can return extra data in bindings.
+### Changes Since v2.12
```
Comment (avade) [inline spec.md]: Again, links here to the feature would be very helpful.

**Comment by avade** [comment]
Looks like you missed updating the ToC to `changes since v2.12` https://github.com/mattmcneeney/servicebroker/blob/c6cd947f6384a90db26e072004bca6413e0bf79f/spec.md#table-of-contents

**Comment by mattmcneeney** [comment]
Thanks @avade - fixing now

File: spec.md (hunk 2 above)
Comment (mattmcneeney) [reply spec.md]: done

File: release-notes.md (hunk 1 above)
Comment (mattmcneeney) [reply release-notes.md]: done

**Comment by mattmcneeney** [comment]
@angarg12 Yep links won't work until the `v2.13` branch is created (I can do this now if people would prefer so we can test this?). I've tested the links replacing `v2.13` with `master` and they all work.

**Comment by angarg12** [comment]
@mattmcneeney that's ok as long as we are aware of the fact :). However in the future it may be distracting that the build fails, making it harder to detect genuine failures. We can discuss it if it is relevant for other people.

**Comment by duglin** [comment]
LGTM

**Comment by avade** [comment]
LGTM now thanks!

**Comment by zrob** [comment]
LGTM

**Comment by mattmcneeney** [comment]
One more LGTM needed, but **do not merge until after Tuesday 26th September** as per the review process.

**Comment by angarg12** [comment]
LGTM

**Comment by vaikas-google** [comment]
LGTM

**Comment by jmrodri** [comment]
Great, this is exciting. LGTM

**Comment by mattmcneeney** [comment]
I squeezed in #222 so this needs another round of LGTMs...

[Approved with PullApprove](https://pullapprove.com/openservicebrokerapi/servicebroker/pull-request/318/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=openservicebrokerapi/servicebroker)

**Comment by vaikas-google** [comment]
LGTM

**Comment by avade** [comment]
lgtm - I think it's important that was included in this release @mattmcneeney - helps out the bluemix folks. Nice one.

**Comment by duglin** [comment]
LGTM

**Comment by shalako** [comment]
LGTM

**Comment by mattmcneeney** [comment]
Creating the release now! 🎉

**Pull request merged**